battle.cpp

```diff
@@ -686,6 +713,12 @@ void Game::GetSides(ARegion *r, AList &afacs, AList &dfacs, AList &atts,
 				}
 
 				if (add == ADD_ATTACK) {
+					if (extendFortDefence && !fortDefenceCleared) {
+						// EXTENDED_FORT_DEFENCE_COVERAGE is not working for attacker garrisons
```

Comment:
I think we should keep same behavior for attack and defense so let's remove this check and add fort defenses for attacking too.

Reply:
Makes sense.